index.html

```diff
@@ -43,7 +43,7 @@ <h1>Table demo</h1>
   <h2>Example content</h2>
   <p>The table:</p>
   <table>
-    <tr><th colspan=3 data-colwidth="100,0,0">Wide header</td></tr>
+    <tr><th colspan=3 data-colwidth="100,0,0"><td>Wide header</td></th></tr>
```

Comment:
```suggestion
    <tr><th colspan=3 data-colwidth="100,0,0">Wide header</th></tr>
```

No need to add a `td` inside if you could just close the `th` instead.